solr/CHANGES.txt

```diff
@@ -109,8 +109,11 @@ New Features
 
 Improvements
 ---------------------
+* SOLR-17158: Users for whom partial results are uninteresting may set partialResultsAllowed=false. This allows Solr to reduce time spent processing partial results and omit them from the response. (Gus Heck, Andrzej Bialeki, hossman)
```

Comment:
Typo - my last name is Bialecki :) or actually Białecki where non-ascii is allowed, but I've been using both forms.

Reply:
uninteresting, undesirable, unacceptable, aren't acceptable .... any of those seem roughly equivalent to me. The point is the user doesn't want them. For example a legal discovery search that omits some results could cause a user to miss an important precedent and loose a case. They would set this parameter to avoid wasting bandwidth, and also forestall the possibility that badly coded UI could show incomplete results get used for an operation requiring complete results.

Reply:
If partial results are not allowed but the timeAllowed limit is reached, is the search effectively cancelled altogether then?  I recall this has been the case with timeAllowed & partialResults=false param (`CommonParams.PARTIAL_RESULTS`) already.  If this CHANGES.txt entry is trying to communicate the objective of this PR, I'm very confused and I think other users might be too, who I thought have already leveraged this behavior (or thought they were).